src/Symfony/Component/HttpFoundation/ParameterBag.php

```diff
@@ -105,40 +105,73 @@ public function remove(string $key)
      */
     public function getAlpha(string $key, string $default = ''): string
     {
-        return preg_replace('/[^[:alpha:]]/', '', $this->get($key, $default));
+        return preg_replace('/[^[:alpha:]]/', '', $this->getString($key, $default));
```

Comment:
all those changed call can lead to UnexpectedValueException
but InputBag is expected to throw BadRequestException instead
to fix this, can't we use filter maybe? I'm looking for a way that wouldn't require wrapping all methods :)

Reply:
yes, my bad :)